WooCommerce/Classes/POS/Utils/POSShadowStyle.swift

```diff
@@ -45,14 +80,79 @@ extension View {
             .frame(width: 200, height: 100)
             .foregroundStyle(Color.posOnSecondaryContainer)
             .background(Color.posOutlineVariant)
-            .posShadow(.medium)
+            .posShadow(.medium, cornerRadius: 12)
 
         Text("Large Shadow")
             .padding()
             .frame(width: 200, height: 100)
             .foregroundStyle(Color.posOnSecondaryContainer)
             .background(Color.posOutlineVariant)
-            .posShadow(.large)
+            .posShadow(.large, cornerRadius: 16)
     }
     .padding(100)
 }
+
+// MARK: - UIKit Implementation (Private)
+
+/**
+ UIKit Implementation for POS Shadows
+ -----------------------------------
+
+ Instead of stacking multiple SwiftUI `.shadow` modifiers (which each trigger a separate offscreen
+ render pass and can be very expensive in lists or scrolling views), this approach uses a custom UIView (`MultiShadowView`)
+ that adds multiple CALayer sublayers, each with its own shadow configuration.
+
+ Performance Benefits:
+ - All shadows are rasterized and composited in a single view hierarchy, reducing the number of offscreen passes.
```

Comment:
Along with the above comment (if that's correct) then we should change this too:

```suggestion
 - All shadow operations are rasterized and composited using a single CALayer in a single view hierarchy
```